js/src/Map.js

```diff
@@ -244,6 +278,17 @@ export class LeafletMapView extends utils.LeafletDOMWidgetView {
       this.create_panes();
       this.layer_views.update(this.model.get('layers'));
       this.control_views.update(this.model.get('controls'));
+      var layer_list = this.model.get('layers');
+      var all_subitems = [];
+      layer_list.forEach((layer) => {
+        var subitem_list = layer.attributes.subitems;
+
+        subitem_list.forEach((subitem) => {
+          all_subitems.push(subitem);
+        });
```

Comment:
This is equivalent to:

```suggestion
        all_subitems = all_subitems.concat(subitem_list)
```